pkg/pipeline/conntrack/hash.go

```diff
@@ -65,8 +66,11 @@ func ComputeHash(flowLog config.GenericMap, keyFields api.KeyFields) ([]byte, er
 func computeHashFields(flowLog config.GenericMap, fieldNames []string) ([]byte, error) {
 	h := fnv.New32a()
 	for _, fn := range fieldNames {
-		// TODO: How should we handle a missing fieldName?
-		f := flowLog[fn]
+		f, ok := flowLog[fn]
+		if !ok {
+			log.Warningf("Missing field %v", fn)
```

Comment:
This could be risky.  It could end up flooding the log file.  Is there a way to avoid repeated logs?

Reply:
We can replace it with a prometheus metric. WDYT?

Reply:
I don't believe this needs to be tracked as a metric.  It doesn't seem like it's likely to occur but can if someone wants to be malicious.  For now, maybe let it go, but we should have a general solution to avoid spamming the log file.